pkg/views/common.go

```diff
@@ -46,6 +47,25 @@ var (
 	widthBreakpoints = []int{60, 80, 100, 120, 140, 160}
 )
 
+// Resources that have actions being performed on them have a higher priority when listing
+var ResourceListStatePriorities = map[apiclient.ModelsResourceStateName]int{
+	apiclient.ResourceStateNamePendingCreate:       1,
+	apiclient.ResourceStateNamePendingStart:        1,
+	apiclient.ResourceStateNamePendingStop:         1,
+	apiclient.ResourceStateNamePendingRestart:      1,
+	apiclient.ResourceStateNamePendingDelete:       1,
+	apiclient.ResourceStateNamePendingForcedDelete: 1,
+	apiclient.ResourceStateNameCreating:            1,
+	apiclient.ResourceStateNameStarting:            1,
+	apiclient.ResourceStateNameStopping:            1,
+	apiclient.ResourceStateNameDeleting:            1,
+	apiclient.ResourceStateNameStarted:             2,
+	apiclient.ResourceStateNameUndefined:           2,
+	apiclient.ResourceStateNameError:               3,
+	apiclient.ResourceStateNameUnresponsive:        4,
+	apiclient.ResourceStateNameStopped:             5,
+}
```

Comment:
If there are going to be only 5 priority levels, perhaps we should create constants e.g. const VERY_HIGH = 1, HIGH = 2, etc.
Reasoning for this is that there might rise an issue where these should be used once again and two different developers might observe priority levels differently by default.
This way, it's immediately clear which integer represents which level of priority.

Reply:
We discussed that we should keep the numbers since it would allow for easier changes in the future.